docs/source/faq/index.rst

```diff
@@ -1,8 +1,13 @@
+
+.. _faq:
+
 Frequently Asked Questions
 ==========================
 
 .. default-role:: code
 
+.. _faq-browser-support:
```

Comment:
Staring a convention where we provide anchor links to all FAQ questions that begin with `faq`, so that we can link to them as necessary

Reply:
I like it, that was pretty handy when linking to some of the docs you've done this for previously